profile/README.md

```diff
@@ -0,0 +1,22 @@
+![Containers](https://github.com/containers/.github/blob/main/logo/containers-logo.png)
+
+# Welcome to Containers! 
+We're a collection of open source tools that create, configure, and work with containers.
+
+## 🛠️ Tools
+Some of our tools include [Podman](https://github.com/containers/podman), [Podman Desktop](https://github.com/containers/podman-desktop), [Buildah](https://github.com/containers/buildah), [Skopeo](https://github.com/containers/skopeo), [conmon-rs](https://github.com/containers/conmon-rs), and [crun](https://github.com/containers/crun), just to name a few. We also have several supporting libraries for these tools, such as [common](https://github.com/containers/common), [storage](https://github.com/containers/storage), [image](https://github.com/containers/image), and more. Check them out and see all the cool things happening!
+
+## 🤝 Community
+We have an active community. Most of our community communication channels are around [Podman](https://github.com/containers/podman), but we definitely welcome anyone with an interest in containers or any of our tools.
+
+Maintainers of Containers and other community members generally chat here:
+- IRC: #podman on [libera.chat](https://libera.chat/)
+- Matrix: [#podman:matrix.org](https://matrix.to/#/#podman:fedoraproject.org)
+- Discord: [Podman](https://discord.com/invite/x5GzFF6QH4)
+
+The Matrix room has been bridged with the Discord server and the libera.chat channel so joining one should be sufficient.
+
+We hold also [community meetings](https://podman.io/community/#community-meetings) around once a month and have a [mailing list](https://podman.io/community/#mailing-list) for additional discussion.
+
+## 👩‍💻 Contributing
+We welcome and encourage contributions, including issues, discussions, and PR's. Please make sure to read our [Contributing Guidelines](https://github.com/containers/common/blob/main/CONTRIBUTING.md) and our [Code of Conduct](https://github.com/containers/common/blob/main/CODE-OF-CONDUCT.md) before doing so.
```

Comment:
I'd add a Security section with a note to check each project's security.md in their top-level repository.  The security.md is used by 95%+ of the projects, but one or two went and did their own thing.

Reply:
Can you include a "We are!" type of section which just includes the little shields for each project that has them?  then if you click on the shield, it would take you to the GitHub repo for that project?  Preferabbly all the shields lined up horizontally  i.e.  O O O
rather than vertically
O
O
O

Reply:
On the fence, should we drop the YouTube channel link in here?  I'm on the fence.  Speaking of which, I don't think I've a link to that in podman.io or podman and probably should.

Reply:
@TomSweeneyRedHat  I think we could leave the youtube channel off here, since its mostly community meetings and not like, produced content.

I have a section on our tools a few lines above, do you think the shields should go there? Or have its own separate section. From what I've seen we have logos for podman, buildah, skopeo, and podman-desktop

Reply:
Okay, fixed everything else, shields I think are coming in a later PR, as the shields atm are not margin-normalized, so it looks very wonky. I've reached out to Mo already :)